src/Tests/Cluster/NodesUsage/NodesUsageUnitTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using FluentAssertions;
+using Tests.Framework;
+
+namespace Tests.XPack.DeprecationInfo
+{
+	public class NodesUsageUnitTests
+	{
+		[U]
+		public void ShouldDeserialize()
```

Comment:
Not sure this is needed; `ExpectResponse` in `NodesUsageApiTests ` is also testing deserialization?

Reply:
This is a fuller example.